packages/test/test-end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -734,7 +734,7 @@ describeFullCompat("Detached Container", (getTestObjectProvider) => {
 		await defPromise.promise;
 	});
 
-	it("Load attached container from cache and check if they are same", async () => {
+	it.skip("Load attached container from cache and check if they are same", async () => {
```

Comment:
These tests should not be skipped. Cache is still supported, its only disabled by default which means we should test it.
You should explicitly enable cache in this test.

Reply:
do I enable it by passing the headers? but what happens when I eventually remove the headers?

Reply:
When you remove the headers, you have to reason about whether we still need this coverage or if its okay to remove the test. This particular test is about loader cache, if there is no cache, the test doesn't make sense so should be removed. You have to do this analysis for each test.